library/Zend/Form/Element/DateSelect.php

```diff
@@ -89,7 +89,7 @@ public function getDayAttributes()
     /**
      * @param  string|array|\ArrayAccess|PhpDateTime $value
      * @throws \Zend\Form\Exception\InvalidArgumentException
-     * @return void|\Zend\Form\Element
+     * @return DateSelect
```

Comment:
@samsonasik  The other setters in this class use @return DateSelect
would you like me to update this and the other setter docblocks to show return self ?